pkg/providers/instance/instance.go

```diff
@@ -108,10 +108,6 @@ func (p *Provider) Link(ctx context.Context, id, provisionerName string) error {
 				Key:   aws.String(v1alpha5.ManagedByLabelKey),
 				Value: aws.String(settings.FromContext(ctx).ClusterName),
 			},
-			{
-				Key:   aws.String(fmt.Sprintf("kubernetes.io/cluster/%s", settings.FromContext(ctx).ClusterName)),
-				Value: aws.String("owned"),
-			},
```

Comment:
Why remove the cluster tag?

Reply:
This is for CreateTag tagging after the fact on the migration path. If we want to scope the permission such that Karpenter can't modify the cluster tag on instances that don't already belong to the cluster, then we have to assert that this tag already exists on the instance. In that case, it's superfluous to modify this tag since we know it's already there.

Reply:
👍